qemu/qmp_test.go

```diff
@@ -1221,3 +1221,20 @@ func TestExecuteVirtSerialPortAdd(t *testing.T) {
 	q.Shutdown()
 	<-disconnectedCh
 }
+
+// Checks migration status
+func TestExecuteQueryMigration(t *testing.T) {
+	connectedCh := make(chan *QMPVersion)
+	disconnectedCh := make(chan struct{})
+	buf := newQMPTestCommandBuffer(t)
+	buf.AddCommand("query-migrate", nil, "return", nil)
```

Comment:
Could we modify the test case so that it returns some valid data?  This would then check the marshaling code is working correctly.  There's an example of another test case that does this here:

https://github.com/intel/govmm/blob/7f8997b949df435ca640fa59298cd575aa103b74/qemu/qmp_test.go#L1045